src/main.ts

```diff
@@ -181,20 +167,24 @@ export default class ShikiPlugin extends Plugin {
 
 	async loadSettings(): Promise<void> {
 		this.settings = Object.assign({}, DEFAULT_SETTINGS, await this.loadData()) as Settings;
+
+		// migrate the theme to darkTheme and lightTheme
+		if (this.settings.theme !== undefined) {
+			this.settings.darkTheme = this.settings.theme;
+			this.settings.lightTheme = this.settings.theme;
+			this.settings.theme = undefined;
```

Comment:
prefer `delete this.settings.theme`. Setting the value to undefined leaves it in the `data.json` file.